htdocs/compta/facture/class/facture.class.php

```diff
@@ -3347,8 +3347,17 @@ function getNextNumRef($soc,$mode='next')
 
 			$mybool=false;
 
-			$file = $conf->global->FACTURE_ADDON.".php";
-			$classname = $conf->global->FACTURE_ADDON;
+			
+			
+			$constant = 'FACTURE_ADDON_'.$this->entity;
+			
+			if (! empty($conf->global->$constant)) {
+				$classname = $conf->global->$constant; // for multicompany proposal sharing
+			} else {
+				$classname = $conf->global->FACTURE_ADDON;
+			}
+			$file = $classname.".php";
```

Comment:
@atm-quentin 
you can revert this, with multicompany 9 this constant is override by multicompany module

Reply:
great, i do it